File: holiday_warmups/removeSpaces.js
const removeSpaces = (strInput) => {
  //   strInputArray = [];
  var strInputArray = strInput.split(" ");
  //   console.log(strInputArray);
  //   strInputArray.forEach((str) => {
  //     // console.log(str);
  //     if (str == " ") {
  //       console.log("this is working");

  //       strInputArray.pop(" ");
  //       console.log(strInputArray);
  //     }
  //   });
  return strInputArray.join("");
};

console.log(removeSpaces("cake pudding"));
console.log(removeSpaces(" cake pudding "));
console.log(removeSpaces("cake     pudding    "));
console.log(removeSpaces("p u d d i n g"));

File: holiday_warmups/removeSpaces.rb
def remove_spaces (str_input) 

    str_input_array = str_input.split(' ')
    return str_input_array.join('')

end

p(remove_spaces("cake pudding"));
p(remove_spaces(" cake pudding "));
p(remove_spaces("cake     pudding    "));
p(remove_spaces("p u d d i n g"));